EXAMPLES.md

```diff
@@ -370,13 +378,18 @@ On failure, you will receive a `ShopifyError`. There are 3 types of `ShopifyErro
 
 `HTTP` errors will be thrown when there was an issue connecting or downloading to a required web server.
 
-`NativePaymentProcessingError` will be thrown when Apple Pay fails to generate a token while trying to authenticate the user's card. This error is unrecoverable and you should fall back to a different payment method, or allow the user to try going through the process again.
+`NativePaymentProcessingError`
+
+This error is thrown whenever something goes wrong on the native pay side of the checkout. It is unrecoverable and you should fall back to a different payment method, or allow the user to try going through the process again. The reasons for this error to be thrown depend on the platform:
+
+* **iOS:** will be thrown when Apple Pay fails to generate a token while trying to authenticate the user's card.
+* **Android:** will be thrown for any unrecoverable Android Pay error that is prefixed with `ERROR_` ([see the full list](https://developers.google.com/android/reference/com/google/android/gms/wallet/WalletConstants)), such as _service unavailable_, _unsupported country_ or _authentication error_.
```

Comment:
I would reword the end of this sentence to say:

`You can see a full list of there errors [here](...)`

Reply:
https://github.com/Shopify/unity-buy-sdk/pull/362/commits/4a75989ae3c72b5698fd85a8b1ffdaedfc606f3d